packages/react-core/src/components/TreeView/TreeViewList.tsx

```diff
@@ -18,11 +23,35 @@ export const TreeViewList: React.FunctionComponent<TreeViewListProps> = ({
   isNested = false,
   onSearch,
   searchProps,
+  toolbarItems,
+  toolbarLeft,
   children,
   ...props
 }: TreeViewListProps) => (
   <>
-    {onSearch && (
+    {toolbarItems && onSearch && toolbarLeft && (
+      <React.Fragment>
+        <Toolbar>
+          <ToolbarContent>
+            {toolbarItems}
```

Comment:
In a `Toolbar` each item needs to be wrapped in a `ToolbarItem` component.

Reply:
@nicolethoen, I'm not following you as we don't have control of the `toolbarItems` content, so how could we handle its details and break it down by items.

Reply:
Possibly have the items passed in an array and loop over the array and wrap each item in a `ToolbarItem` component?
That is just how our structure and styles for `Toolbar` are applied according to the design specs. 
https://www.patternfly.org/v4/components/toolbar#items

Reply:
We could do that or just require that the consumer put `<ToolbarItem>` around them all themselves, in case there is some custom JSX they need in the future with a variant of `<ToolbarItem>` or a special className on it or something. The consumer has to wrap their items that way in normal Toolbar usage anyway.

Reply:
that'd be fine- it'd just need to be included in the docs